src/config.js

```diff
@@ -3,6 +3,9 @@ const config = {
     // release3: 'https://raw.githubusercontent.com/commaai/openpilot/release3/system/hardware/tici/agnos.json',
     master: 'https://raw.githubusercontent.com/commaai/openpilot/master/system/hardware/tici/agnos.json',
   },
+  loader: {
+    url: "https://raw.githubusercontent.com/bkerler/Loaders/main/qualcomm/factory/sdm845_sdm850_sda845/6000000000010000_f8ab20526358c4fa_fhprg.bin",
```

Comment:
let's copy this here and link from the repo

Reply:
Updated, but commented out, uncomment when ready to merge